spec/lib/newspaper_works/ingest/ndnp/page_metadata_spec.rb

```diff
@@ -57,4 +57,28 @@
       expect(page.identifier).to eq "././0225.tif"
     end
   end
+
+  describe "sample fixture via Reel XML" do
+    let(:page) { described_class.new(reel1, nil, 'targetModsBib1') }
+
+    it "returns nil page number for page without one" do
+      expect(page.page_number).to eq nil
+    end
+
+    it "gets expected sequence number as Integer" do
```

Comment:
I would expect this spec to be checking for the existence of an Integer value, not `nil`?

Reply:
This test needs a review.  There is no sequence number for a page in reference to a reel; I feel it should be nil, here — but if that actually makes sense then I need to adjust the spec description, clearly.